src/assets/scss/app.scss

```diff
@@ -64,7 +64,12 @@
 @include motion-ui-animations;
 
 $message-new: $white;
+$message-new-text: #404040;
 $message: $body-background;
+$message-text: darken($message, 20);
+
+$font-sizes: (1rem, 0.9rem, 0.85rem);
+$color-collection: ('message': '#2274D5');
```

Comment:
jinak zase zapremejslel nadtím, jestli by nebylo zajímavější mít jednu proměnnou, která by obsahovala všechny barvičky , které kompoennta využívá a z ní bys tahal co bude třeba..

Reply:
zkusil jsem to, teda dal jsem to do 2. jedna pro ikony a druha pro zbytek stranky. Nevim zda map-get je optimalni pro dostavani hodnot. Nenasel jsem zadnou alternativu